lib/src/client.dart

```diff
@@ -351,6 +351,21 @@ class EOSClient {
 
     return PushTransactionArgs(signatures, serializedTrx);
   }
+
+  Future<AccountNames> getKeyAccountsState(String pubKey) async {
+    http.Response response = await http.post(
+        '${this._nodeURL}/${this._version}/state/get_key_accounts',
+        body: {"public_key": pubKey});
+
+    if (response.statusCode >= 300) {
+      throw response.body;
```

Comment:
can we do better instead of throw the body?

Reply:
Finnaly, I don't need this anymore so I'll remove it from the pull request